tftp/util.py

```diff
@@ -41,8 +44,7 @@ def iterate(_=None):
             else:
                 return deferLater(clock, delay, call).addCallback(iterate)
         else:
-            # No timings were given.
-            return succeed(None)
+            raise ValueError("No timings specified.")
```

Comment:
Isn't it better to raise as soon as possible here? That is, in the outer function (`timedCaller`)? Unless I am missing something, there's nothing gained by moving this check down to `iterate`